README.NL.rst

```diff
@@ -41,6 +41,8 @@ latest          n.v.t.          `ReDoc <https://redocly.github.io/redoc/?url=htt
                                 (`verschillen <https://github.com/maykinmedia/objects-api/compare/1.0.0..master#diff-b9c28fec6c3f3fa5cff870d24601d6ab7027520f3b084cc767aefd258cb8c40a>`_)
```

Comment:
Change latest diff to compare against 1.1.0

Reply:
done, and I removed the hash from the diff